### ISSUE TITLE
Users missing "Unregistered" tag

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2050]
Version: 2.0.3-alpha.2 
mobile@2.0.3-alpha.2 (iOS 329)

While fixing issue #119 something was broken and new users that are joining while the Owner is offline doesn't get "Unregistered" tag. "Duplicate" tag works as expected.

Steps to recreate:
1. Join to the community while the Owner is offline.
2. Send a message.
What happens - user can send a message, username doesn't have "Unregistered" tag.
What should happen - user can send a message, username have "Unregistered" tag.

**Comment by siepra** [comment]
Isn't it a duplicate of https://github.com/orgs/TryQuiet/projects/3/views/1?pane=issue&itemId=39490025 @kingalg ?

**Comment by kingalg** [comment]
It looks like it is, so I'm closing this one. Strange thing is that I'm pretty sure that it was working in some versions between - I've opened initial issue almost 2 months ago and I saw some Unregistered tags since then. So there may be some other reason behind the problem. But I wouldn't bet my life on it. I was mostly focused on duplicate users recently, I might have imagine things.

Closing as duplicate.
